README.md

```diff
@@ -83,7 +83,28 @@ val key = Secrets().getYourSecretKeyName(packageName)
 final String key = new Secrets().getYourSecretKeyName(getPackageName());
 ```
 
-# 4 - (Optional) Improve your key security
+# 4.1 - (Optional) Generate secrets from properties file
```

Comment:
I think this new behaviour needs is own section so I propose you set it after the section 4, something like 
`# Going further - (Optional) Generate secrets from properties file`